stoqs/loaders/CANON/loadCANON_os2015.py

```diff
@@ -124,25 +124,37 @@ def find_urls(base):
 # the binned files before this will work
 
 # Get directory list from thredds server
-#platforms = ['tethys', 'daphne', 'makai']
-platforms = ['makai', 'tethys']
+platforms = ['tethys', 'daphne', 'makai']
 
 for p in platforms:
     base =  'http://elvis.shore.mbari.org/thredds/catalog/LRAUV/' + p + '/missionlogs/2015/'
     dods_base = 'http://dods.mbari.org/opendap/data/lrauv/' + p + '/missionlogs/2015/'
     setattr(cl, p + '_files', [])
     setattr(cl, p + '_base', dods_base)
-    setattr(cl, p + '_parms' , ['temperature', 'salinity', 'chlorophyll', 'nitrate', 'oxygen','bbp470', 'bbp650','PAR'])
-    urls = find_urls(base)
+    setattr(cl, p + '_parms' , ['temperature', 'salinity', 'chlorophyll', 'nitrate', 'oxygen','bbp470', 'bbp650','PAR'
+                                'yaw', 'pitch', 'roll', 'control_inputs_rudder_angle', 'control_inputs_mass_position',
+                                'control_inputs_buoyancy_position', 'control_inputs_propeller_rotation_rate',
+                                'health_platform_battery_charge', 'health_platform_average_voltage',
+                                'health_platform_average_current','fix_latitude', 'fix_longitude',
+                                'fix_residual_percent_distance_traveled_DeadReckonUsingSpeedCalculator',
+                                'pose_longitude_DeadReckonUsingSpeedCalculator',
+                                'pose_latitude_DeadReckonUsingSpeedCalculator',
+                                'pose_depth_DeadReckonUsingSpeedCalculator',
+                                'fix_residual_percent_distance_traveled_DeadReckonUsingMultipleVelocitySources',
+                                'pose_longitude_DeadReckonUsingMultipleVelocitySources',
+                                'pose_latitude_DeadReckonUsingMultipleVelocitySources',
+                                'pose_depth_DeadReckonUsingMultipleVelocitySources'])
+    urls_eng = find_urls(base, '.*2S_eng.nc$')
+    urls_sci = find_urls(base, '.*10S_sci.nc$')
+    urls = urls_eng + urls_sci
     files = []
     if len(urls) > 0 :
         for url in sorted(urls):
             file = '/'.join(url.split('/')[-3:])
             files.append(file)
-    #files.append(',')
     setattr(cl, p + '_files', files)
     setattr(cl, p  + '_startDatetime', startdate)
-    setattr(cl, p + '_endDatetime', enddate)
+    setattr(cl, p + '_endDatetime', enddate))
```

Comment:
Yes, I know that’s a typo I found yesterday.  Thanks for the note. I’ll try to get the fixed today.

> On Feb 3, 2016, at 9:52 AM, Mike McCann <notifications@github.com> wrote:
> 
> In stoqs/loaders/CANON/loadCANON_os2015.py <https://github.com/stoqs/stoqs/pull/245#discussion_r51759152>:
> 
> >      setattr(cl, p + '_files', files)
> >      setattr(cl, p  + '_startDatetime', startdate)
> > -    setattr(cl, p + '_endDatetime', enddate)
> > +    setattr(cl, p + '_endDatetime', enddate))
> Landscape.io complains about the extra ')' on this line:
> 
> https://landscape.io/github/stoqs/stoqs/36/modules/stoqs/loaders/CANON/loadCANON_os2015.py <https://landscape.io/github/stoqs/stoqs/36/modules/stoqs/loaders/CANON/loadCANON_os2015.py>
> —
> Reply to this email directly or view it on GitHub <https://github.com/stoqs/stoqs/pull/245/files#r51759152>.
>